check.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash -e
-dep ensure
+#dep ensure
 #gometalinter --skip test --skip defaults --cyclo-over=15 --deadline=90s --vendor ./...
-golangci-lint run
+go mod tidy
+#golangci-lint run
```

Comment:
Clobbered linter

Reply:
Ahh :(
I'll fix that. Didn't notice right away.